src/controllers/productController.js

```diff
@@ -0,0 +1,51 @@
+import { product } from '../database/models';
+
+const getProductById = async (req, res) => {
+  console.log('hello');
+  try {
+    const { id } = req.params;
+    const specificItem = await product.findOne({
+      where: { id },
+    });
+
+    if (!id) {
+      return res.status(404).json({
+        status: 'fail',
+        message: 'Missing id parameter in request.',
```

Comment:
Could you use translation on all the messages returned in the API response.

Reply:
good work @nellykawera  you putted your effort in this project  but it's also a good practice to not push console's, so i would suggest you to remove them